chapter04/pxd/ex3.pl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use feature "say";
+use Scalar::Util qw( looks_like_number );
+
+sub total {
+    chomp(my @vars=@_ );
+    my $total=0;
+
+    foreach my $number (@vars) {
+        if ( looks_like_number ($number)) {
+            $total += $number;    
+        } else {
+            say " {$number} is not a numeric --> INGORING!! ";
+        }
+    }
+    return $total;
+}
+
+
+sub average {
+    return  @_ ? total(@_)/scalar @_ : 0;
```

Comment:
Oh, cool. One line! Mine average function has 7 lines... 
BTW, it may work without the scalar there?